packages/docs-page/index.tsx

```diff
@@ -48,7 +65,8 @@ export function DocsPageWrapper({
     <SearchProvider>
       <SearchBar
         product={name}
-        className={classNames({ [s.mobileSearch]: isMobile })}
+        // TODO; SearchBar doesn't accept `className`
+        // className={classNames({ [s.mobileSearch]: isMobile })}
```

Comment:
Old bug?

Reply:
I think this might have been a casualty of the CSS modules migration. `SearchBar` here doesn't accept a `className`.

@thiskevinwang we should be update to update `SearchBar` to accept `className` and pass it on to the underlying search. https://github.com/hashicorp/react-components/blob/main/packages/docs-page/components/search-bar/index.jsx#L8

Reply:
Will update!